packages/loaders/graphql-file/src/index.ts

```diff
@@ -110,7 +110,11 @@ export class GraphQLFileLoader implements Loader<GraphQLFileLoaderOptions> {
   }
 
   async load(pointer: string, options: GraphQLFileLoaderOptions): Promise<Source[]> {
-    const resolvedPaths = await this.resolveGlobs(pointer, options);
+    let resolvedPaths: string[] = [];
+    if (!pointer.includes('*') && await this.canLoad(pointer, options))
+      resolvedPaths = [pointer]
+    else
+      resolvedPaths = this.resolveGlobsSync(pointer, options);
```

Comment:
Is there any reason to use `sync` instead of the other one?

Reply:
Thanks for the review.
My mistake, should be the other one for line 117. 
```const resolvedPaths = await this.resolveGlobs(pointer, options);```

Reply:
Will submit a revision now